javascript/packages/orchestrator/src/configGenerator.ts

```diff
@@ -42,6 +42,20 @@ import {
   envVars,
 } from "./types";
 
+const default_keystore_key_types: string[] = [
```

Comment:
We can move this list to `const` an reuse here and https://github.com/paritytech/zombienet/pull/1235/files#diff-b1dd06ee7d2b94fe91604cba031f78c09a937a5f71cd659fa4baf7601ac9ecfbR94, so we maintain only one list of possible keys.